src/systemctl/systemctl.c

```diff
@@ -3987,6 +3987,38 @@ static void print_status_info(
         else
                 printf("\n");
 
+        if (endswith(i->id, ".timer")) {
+                dual_timestamp nw;
+                char *unit_path;
+                char tstamp1[FORMAT_TIMESTAMP_RELATIVE_MAX], tstamp2[FORMAT_TIMESTAMP_MAX];
+                char *next_rel_time = NULL, *next_time = NULL;
+
+                printf("  Trigger: ");
+
+                dual_timestamp_get(&nw);
+                unit_path = unit_dbus_path_from_name(i->id);
+                if (unit_path) {
+                        dual_timestamp next;
+                        usec_t next_elapse;
+
+                        r = get_next_elapse(bus, unit_path, &next);
```

Comment:
invoking get_next_elapse() here means a second bus call to acquire this info. However, we actually already got this info delivered in the earlier GetAll() call, it's juts a matter of actually making use of it in the `status_property()` function, and adding it to UnitStatusInfo.